biomage/rds/run.py

```diff
@@ -54,6 +54,11 @@ def run(command, sandbox_id, input_env, user, region):
         biomage rds run psql\n
         biomage rds run pg_dump > dump.sql
     """
+
+    run_rds_command(command, sandbox_id, input_env, user, region)
```

Comment:
Refactored so that it can be used in other files. The function `run` can not be directly imported and used because it is wrapped by the `click` decorators.